pkgs/by-name/li/library/package.nix

```
@@ -0,0 +1,93 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+  gallery-dl,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "library";
+  version = "2.9.036";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "chapmanjacobd";
+    repo = "library";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-kWL8MkXmKGu78ztuZuJbf3+iufMR2+br0I021ITVd5k=";
+  };
+
+  # project has similar problem
+  # fix https://github.com/pypa/installer/issues/194
+  patches = [ ./fix-duplicate.patch ];
+
+  prePatch = ''
+    # doesn't exists in source distribution
+    sed -i "/^license.=\|^readme.=/d" pyproject.toml
+  '';
+
+  build-system = [ python3.pkgs.hatchling ];
+
+  dependencies = with python3.pkgs; [
```

```suggestion
  dependencies = with python3Packages; [
```
I think python3Packages is preferred.